docs/api/conf.py

```diff
@@ -19,7 +19,7 @@
 
 sys.path.insert(0, root_dir)
 
-exec(open(root_dir / "cloud_registry" / "version.py").read())
+__version__ = open(root_dir / "cloud_registry" / "version").read()
```

Comment:
I've recently used markdown-based API reference docs and find them more satisfying than this stuff + Read the Docs, in many ways.

Of course, for now update the version parsing. Just wanted to give a heads up that we'll probably replace this with a different mechanism soon.